spec/lib/workflow/task-graph-spec.js

```diff
@@ -153,6 +153,17 @@ describe('Task Graph', function () {
             return Promise.all([p1, p2]);
         });
 
+        it('should not fail option validation on falsey option values', function() {
+            definitions.testTask.options.option1 = 0;
+            definitions.graphDefinitionInline.tasks[0].taskDefinition.options.option1 = 0;
```

Comment:
It's will be better if the unit-testing not only covers the number `0` but also the Boolean `false` and empty string `''`.